### PR TITLE
[v2int/2.0] Persist sweepTimeoutMs to summary

### PR DESCRIPTION
**Pull request by markfields**
_back-port of #12419 to internal.2.0 release_

## Background

As-is, the Sweep Timeout will always be at least 6 days due to hardcoded Snapshot Cache limit (5 days) and Buffer (1 day). This means it's impossible to test the Sweep Timer and the code that runs after something is SweepReady!

There have been multiple attempts on my part to address this: #11084, #12044, #12331

Much of the difficulty has arisen from the challenge of aligning driver policy with GC config. I'm finally punting on that and tracking it as a separate item
([AB#2238](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/2238)).

## This change

It's pretty simple:

* Persist SweepTimeoutMs in the summary
* For new containers, that can be overridden via the `"Fluid.GarbageCollection.TestOverride.SweepTimeoutMs"` config setting
* Default behavior for new containers (and for backfilling existing containers with no value persisted) is SweepTimeout + 6 days (same as it has been)

### CODE REVIEW

**Review by tyler-cai-microsoft — APPROVED** [review]

File: packages/loader/container-loader/src/containerStorageAdapter.ts [hunk 1]
```diff
@@ -176,7 +176,7 @@ class BlobOnlyStorage implements IDocumentStorageService {
             // some browsers may not populate stack unless exception is thrown
             throw new Error("BlobOnlyStorage not implemented method used");
         } catch (err) {
-            this.logger.sendErrorEvent({ eventName: "BlobOnlyStorageWrongCall" }, err);
+            this.logger.sendTelemetryEvent({ eventName: "BlobOnlyStorageWrongCall" }, err);
```
Comment (agarwal-navin) [inline packages/loader/container-loader/src/containerStorageAdapter.ts:179]: This looks unrelated.

**Review by agarwal-navin — APPROVED** [review]

File: packages/loader/container-loader/src/containerStorageAdapter.ts (hunk 1 above)
Comment (markfields) [reply packages/loader/container-loader/src/containerStorageAdapter.ts:179]: And makes sense anyway - code throwing an error should rarely also log that error. Let the caller decide if it's actually a problem worth recording.
